uint16 index # index of the output which state is requested
---
bool state